node/src/chain_spec/dev.rs

```diff
@@ -27,6 +27,7 @@ pub fn dev_config(
                         get_from_seed::<nimbus_primitives::NimbusId>("Alice"),
                         crate::chain_spec::DEFAULT_STAKING_AMOUNT,
                     )],
+                    crowdloan_fund_pot: 100u128.saturating_add(BASE),
```

Comment:
I think this is ought to be `saturating_mul`. Currently `crowdloan_fund_pot = 10000000100`.
In case this is true, *node/src/chain_spec/mod.rs* needs to be changed as well.

Reply:
Ooppss... My bad.

Thanks @sea212 !

Reply:
I opened #326 so we can take care of this before the PLO.